### DISCUSSION TITLE
Could someone make or post a video about the multiplayer/co-op stuff?

### DISCUSSION THREAD

**Discussion by TjAbodaiVT** [nzp-team/nzportable#638] [General]
See, I'm very stupid, I keep trying to test multiplayer but I don't understand anything. I can't follow written instructions well cause I'm dyslexic so I need instructions visually. It would help me or even other people understand what they are doing. Maybe it's more simple than I'm thinking but I can't figure out how it work and stuff even though it looks simple.

Things I need to know that I think I could read:
- Can Windows versions play with the webgl versions?
- What are all the console commands? (I haven't checked every discussion yet so)

I may add more at a later date, I don't know...

**Comment by TjAbodaiVT** [comment]
So I figured it out but I still don't know about the 2 questions so if you still want to answer, go ahead, but the video should still be done because of dyslexia.

**Reply by GoatCedric** [reply]
seems like the webgl version is broken. 
idk about all console commands, there's over 2000 of them lol

**Comment by MotoLegacy** [comment]
It's not broken.. I hate those assumptions. FTEQW uses holepunching to brute-force p2p play without port forwarding. The consequence is that not all network configurations, routers, etc. are supported. If it doesn't work for you, it means you're victim to that.

**Comment by GoatCedric** [comment]
Ah yes, I did get it to work after downloading the game for windows. Never
got the webgl version to work though, that's the one I was referring to in
the previous email.
Thanks for replying though!

Oh yeah, by the way, it's buggy when trying to revive a teammate, it often
takes a while before the revive process begins.
Thanks again!

Den lör 10 feb. 2024 20:01cypress ***@***.***> skrev:
